tool/teleport/common/teleport.go

```diff
@@ -1017,6 +1022,27 @@ func onIntegrationConfEICEIAM(params config.IntegrationConfEICEIAM) error {
 	return nil
 }
 
+func onIntegrationConfAWSAppAccessIAM(params config.IntegrationConfAWSAppAccessIAM) error {
```

Comment:
Should we move AWS commands to their own file?

Reply:
I can do it in a follow up PR :+1:

Reply:
Sounds good!